spec/requests/v1/books_controller/index_spec.rb

```diff
@@ -39,7 +39,19 @@
         let(:user) { create(:user, user_permission_list: [record_permission]) }
       end
 
-      it_behaves_like '200 OK'
+      describe 'with subtitle' do
+        it_behaves_like '200 OK'
+      end
+
+      describe 'without subtitle' do
+        let(:request) do
+          VCR.use_cassette('retrieve_book_by_isbn_no_subtitle') do
+            get "#{record_url}?isbn=8713638034201"
```

Comment:
@wilco375 isbn_lookup doesn't work, I get a 403 status which probably means the bol.com API token is expired. Can you rotate it and PR it into the `credentials.yml.enc`?

Reply:
Resolved in https://github.com/csvalpha/amber-api/pull/402